api-tests/server/auth_test.go

```diff
@@ -466,12 +482,14 @@ func createAPIKeyWithRole(t *testing.T, name, role string) (int, string) {
 	req.Header.Set("Content-Type", "application/json; charset=utf-8")
 
 	resp, b := doRequest(t, http.DefaultClient, req)
+
 	require.Equalf(t, http.StatusOK, resp.StatusCode, "failed to create API key, status code: %d, response: %s", resp.StatusCode, b)
 
 	var m map[string]interface{}
 	err = json.Unmarshal(b, &m)
 	require.NoError(t, err)
 	apiKey := m["key"].(string)
+	resp.Body.Close()
```

Comment:
I'd move it right after getting `resp` and prepend with `defer`. And to the second `resp` in this func just would give a new name.

Reply:
DOne.